homeworks/andrii.kholodenko_AndriiCold/6-typescript/src/app.ts

```diff
@@ -0,0 +1,103 @@
+import './styles/style.css';
+
+const totalScoreEl = document.querySelector('[data-total-score]') as HTMLHeadingElement;
+const bubbleEl = document.querySelector('[data-bubble]') as HTMLDivElement;
+const dynamicScoreEl = document.querySelector('[data-dynamic-score]') as HTMLDivElement;
+const keyEl = document.querySelector('[data-key]') as HTMLDivElement;
```

Comment:
```suggestion
const totalScoreEl = document.querySelector<HTMLHeadingElement>('[data-total-score]');
const bubbleEl = document.querySelector<HTMLDivElement>('[data-bubble]');
const dynamicScoreEl = document.querySelector<HTMLDivElement>('[data-dynamic-score]');
const keyEl = document.querySelector<HTMLDivElement>('[data-key]');
```
В `querySelector` є дженерік для типізації